lib/chef/application/knife.rb

```diff
@@ -63,7 +63,7 @@ class Chef::Application::Knife < Chef::Application
     :long         => "--disable-editing",
     :description  => "Do not open EDITOR, just accept the data as is",
     :boolean      => true,
-    :default       => false
+    :default => false
```

Comment:
Did this get moved back because it was supposed to line up with the stuff above it, but was off by 1?